pkg/subscriptions/subscriptions.js

```diff
@@ -369,6 +369,26 @@ define([
             update_timeout = window.setTimeout(status_update_failed, 10000);
         }
 
+        function linkify_message(message) {
+            /* expression from on https://github.com/thp/libcontentaction/blob/47350706ecf4290a74245ae11ef76c4eedde729d/data/gen-regexps #
```

Comment:
The question will come up at some point about license. There's no license in that other file. The project itself is 
```Copyright (c) 2009, 2010 Nokia Corporation.``` even though it looks like @mvollmer wrote it.

https://github.com/thp/libcontentaction/blob/47350706ecf4290a74245ae11ef76c4eedde729d/debian/copyright